### PR TITLE
Feature/bbk 568 vital signs

### PR DESCRIPTION
**Pull request by leeming**
These have already been applied on the prod ELK server

### CODE REVIEW

File: rules.disabled/Rejected_print_job.yaml [hunk 1]
```diff
@@ -1,24 +1,22 @@
...
 realert:
-  minutes: 5
+  minutes: 10
 exponential_realert:
   minutes: 60
 filter:
 - query:
     query_string:
-      query: "beat.name:badgemanagement AND message:\"photo upload successful\""
+      query: "bluebadge.message:\"\""
```
Comment (gdhgdhgdh) [inline rules.disabled/Rejected_print_job.yaml:16]: mm? where's the query?

**Review by gdhgdhgdh — CHANGES_REQUESTED** [review]
just the missing query in print_job_rejected_for_60_mins

File: rules.disabled/Rejected_print_job.yaml (hunk 1 above)
Comment (leeming) [reply rules.disabled/Rejected_print_job.yaml:16]: good catch

Comment (leeming) [reply rules.disabled/Rejected_print_job.yaml:16]: Ah, this is the one requirement that was dropped from the ticket. Also note 'rules.disabled' directory. There is currently no query that can resolve this in ELK

Comment (gdhgdhgdh) [reply rules.disabled/Rejected_print_job.yaml:16]: Ah, perfect :)

**Review by gdhgdhgdh — APPROVED** [review]

**Pull request merged**